src/cryptonote_core/blockchain.cpp

```diff
@@ -407,10 +407,6 @@ bool Blockchain::init(BlockchainDB *db, const network_type nettype, bool offline
 	uint64_t top_block_timestamp = m_db->get_top_block_timestamp();
 	uint64_t timestamp_diff = time(NULL) - top_block_timestamp;
 
-	// genesis block has no timestamp, could probably change it to have timestamp of 1341378000...
```

Comment:
Why is this part removed?

Reply:
I assumend this was a left over from monero and the genesis sumokoin block has an time stamp. I am wrong?

Reply:
No, it also has the timestamp set to 0

Reply:
OK this change is wrong, The genesis block has time stamp is zero. I will remove this change.